pkg/remote/terraform/provider.go

```diff
@@ -88,8 +89,14 @@ func (p *TerraformProvider) Runner() *parallel.ParallelRunner {
 }
 
 func (p *TerraformProvider) configure(alias string) error {
+	var version string
+	lockFile, _ := lock.LoadLocksFromFile(".terraform.lock.hcl")
```

Comment:
My first comment is still relevant:

> We should log in debug the diags, it will help us investigate some case where we wont be able to read the lockfile for some reason

We should not log something only when everything is fine, if there is some diags err we should log them in debug too.

Reply:
I did add log messages in the LoadLocksFromFile func https://github.com/cloudskiff/driftctl/blob/4a455a1b9e25cf5ff0e91f3b042fcc6037b7931d/pkg/remote/terraform/lock/lockfile.go#L44-L55

Reply:
My bad I missed this one 🙏🏻 Anyway, I think that we should not bind the lockfile pkg to a logging strategy. Since we log here when everything is fine, it makes more sense to log errors at the same place IMHO. Let's keep the lockfile logic return it's errors without any error handling logic within.